packages/js/product-editor/src/components/attribute-term-input-field/create-attribute-term-modal.tsx

```diff
@@ -41,22 +41,22 @@ export const CreateAttributeTermModal: React.FC<
 	const { createNotice } = useDispatch( 'core/notices' );
 	const [ isCreating, setIsCreating ] = useState( false );
 	const { createProductAttributeTerm, invalidateResolutionForStoreSelector } =
-		useDispatch( EXPERIMENTAL_PRODUCT_ATTRIBUTE_TERMS_STORE_NAME );
+		useDispatch( experimentalProductAttributeTermsStore );
 
 	const onAdd = async ( attribute: Partial< ProductAttributeTerm > ) => {
 		recordEvent( 'product_attribute_term_add', {
 			source: TRACKS_SOURCE,
 		} );
 		setIsCreating( true );
 		try {
-			const newAttribute: ProductAttributeTerm =
-				await createProductAttributeTerm( {
-					...attribute,
-					attribute_id: attributeId,
-				} );
+			const newAttribute = await createProductAttributeTerm( {
+				...attribute,
+				attribute_id: attributeId,
+			} );
 			recordEvent( 'product_attribute_term_add_success', {
 				source: TRACKS_SOURCE,
 			} );
+			// @ts-expect-error TODO: Investigate - is this correct?
 			invalidateResolutionForStoreSelector( 'getProductAttributes' );
```

Comment:
Same as above.

Reply:
I removed it with 1a242d57b2da01527e56addcb9ce0c7fa1485605